tests/integration/test_service_manager.py

```diff
@@ -48,8 +48,8 @@ def test___grpc_service_without_discovery_service___start_service___service_host
 def test___grpc_service_started___stop_service___service_stopped(grpc_service: GrpcService):
     """Test to validate if measurement service is stopped."""
     port_number = grpc_service.start(
-        measurement.measurement_info,
-        measurement.service_info,
+        measurement.sample_measurement_service.measurement_info,
+        measurement.sample_measurement_service.service_info,
```

Comment:
ok, these were already public

Reply:
Should they be?

Reply:
I guess not. The only references to those fields seem to be in this class and its tests. The same is true for `configuration_parameter_list`, `output_parameter_list`, and `grpc_service`.

What about types? `MeasurementInfo` and `ServiceInfo` are used in `DiscoveryService`, which is in an `_internal` module but is used by the example code.

Consider cleaning this up in a separate PR. Don't add it to this one.

Reply:
@bkeryan We're actually documenting these types:
https://ni.github.io/measurementlink-python/autoapi/ni_measurementlink_service/index.html#ni_measurementlink_service.MeasurementInfo

Reply:
I know. We're also documenting these fields: https://ni.github.io/measurementlink-python/autoapi/ni_measurementlink_service/index.html#ni_measurementlink_service.MeasurementService

Reply:
Looks like they're being generated automatically. So I guess I don't need to take any action. If we do hide them, I imagine that'll be reflected in the doc generation as well.